volto/src/addons/volto-chart-builder/src/components/ChartBuilderEdit.jsx

```diff
@@ -18,9 +18,17 @@ const View = () => {
 };
 
 const Edit = (props) => {
-  const { selected, onChangeBlock, block, data } = props;
-  const { error: dataError } = usePloneCsvData(data.file_path || []);
-  const { error: geoJsonError } = usePloneGeoJson(data.geojson_path || []);
+  const { selected, onChangeBlock, block, data, properties } = props;
+
+  const { error: dataError } = usePloneCsvData(
+    properties.parent,
+    data.file_path || [],
+  );
+  // this error does not seem to be being set, despite there being a problem in usePloneCsvData
```

Comment:
can we address this or make sure there's a card too do so please.

Reply:
Made a ticket to investigate this.
https://app.zenhub.com/workspaces/cogs-dd-cms-61e83b63053eb70011d37b4f/issues/gh/gss-cogs/dd-cms/844